### ISSUE TITLE
Schema retrival helper function proposal

### ISSUE THREAD

**Issue by filipeximenes** [bernardopires/django-tenant-schemas#354]
Would a helper function (maybe on the util.py module) be useful to the project? Something like:

```
from django.db import connection

def get_current_tennant():
    return connection.get_tenant()
```
Is there some other way in the project to do this? Is there any problem with this construction?
Something like this would help on things like saving media files in a file according to the tenant it belongs (the upload_to model function does not gets the `request` object)

LMK, I can PR this

**Comment by bernardopires** [comment]
What's the problem with using `connection.get_tenant()`?

**Comment by vinitkumar** [comment]
@filipeximenes I have fixed this easily using what is provided by django-tenant-schemas.

Here is the sample code that would help you achieve this:

```python
from django.db import models, connection

def get_upload_path(instance, filename):
    """
    Returns the upload path for the theming media files, which is in a
    separate folder for clarity.
    """
    return "someapp/%s/%s/%s" % (instance.name,  connection.schema_name, filename)

class Customer(models.Model):
    image = models.FieldField(upload_to=get_upload_path)
```

Here whenever you are going to save the media files, it will save the file with a namspace that is decided by the tenant name. 

cc @bernardopires

**Comment by filipeximenes** [comment]
@bernardopires @vinitkumar All well with the current available solutions. 
To be honest, my real concern is with newcomers (like me) which cannot (IMO) easily find things like this to solve some basic problems that are very common. For example I did a quick search and there's no mention of `get_tenant` method in the documentation. I've reported some problems and missing things on the docs in other issues. I guess this is more about improving docs than actually adding a helper function. 
I've been working with tenant schemas for a little more than a month now and I now I feel confident to do some contribution on the docs. @bernardopires do you mind if I try to do some major reorganization on the docs? I have some experience on docs working on tapioca-wrapper (http://tapioca-wrapper.readthedocs.io/en/stable/) and django-role-permisions (http://django-role-permissions.readthedocs.io/en/latest/)

**Comment by mikicz** [comment]
@filipeximenes If you do that feel free to use some docs for utils I wrote in #356

**Comment by filipeximenes** [comment]
@mikicz Awesome! Your PR covers most of the things I've been worried about :)

**Comment by Scalamoosh** [comment]
+1 for documenting this. Took me a while to find out how to get a tenant with no context of a request.

**Comment by goodtune** [comment]
I've merged #356 so I'll close this ticket, but feel free to create other pull requests for further documentation improvements.
